docs/workWithSheet.md

```diff
@@ -0,0 +1,573 @@
+    "properties": {
+        "title": "Company",
+        "locale": "en_US",
+        "autoRecalc": "ON_CHANGE",
+        "timeZone": "Etc/GMT",
+        "defaultFormat": {
+            "backgroundColor": {
+                "red": 1,
+                "green": 1,
+                "blue": 1
+            },
+            "padding": {
+                "top": 2,
+                "right": 3,
+                "bottom": 2,
+                "left": 3
+            },
+            "verticalAlignment": "BOTTOM",
+            "wrapStrategy": "OVERFLOW_CELL",
+            "textFormat": {
+                "foregroundColor": {},
+                "fontFamily": "arial,sans,sans-serif",
+                "fontSize": 10,
+                "bold": false,
+                "italic": false,
+                "strikethrough": false,
+                "underline": false
+            }
+        }
+    },
+    "sheets": [
+        {
+            "properties": {
+                "sheetId": 789,
+                "title": "Employees",
+                "index": 0,
+                "sheetType": "GRID",
+                "gridProperties": {
+                    "rowCount": 1000,
+                    "columnCount": 26
+                }
+            }
+        }
+    ],
+    "spreadsheetUrl": "https://docs.google.com/spreadsheets/d/1oGxpE3C_2elS4kcCZaB3JqVMiXCYLamC1CXZOgBzy9A/edit"
+}
+```
+
+
+
+###### Related Google Spreadsheet documentation
+
+https://developers.google.com/sheets/reference/rest/v4/spreadsheets/get
+
+
+
+#### Update Sheet Properties Using BatchRequest
+
+This method allows you to update the size, title, and other sheet properties.
+
+
+
+###### updateSheetPropertiesBatchRequest
+
+```xml
+<googlespreadsheet.updateSheetPropertiesBatchRequest>
+      <spreadsheetId>{$ctx:spreadsheetId}</spreadsheetId>
+      <requests>{$ctx:requests}</requests>
+      <fields>{$ctx:fields}</fields>
+</googlespreadsheet.updateSheetPropertiesBatchRequest>
+```
+
+
+
+###### Properties
+
+* spreadsheetId :- Unique value of the spreadsheet
+* requests :- It contains data that is a kind of update to apply to a spreadsheet. To Update multiple sheets properties within the spread sheet, need to repeat `updateSheetProperties` property within the `requests` property.
+* fields (Outside the requests property) [Optional] :- Specifying which fields to include in a partial response. This is define outside the requests body.
+
+
+
+###### Sample request
+
+Here the requests body should contain what are the properties to be changed, and the fields parameter within the requests property should explicitly list those properties (if you want to update all properties, use fields:"*" as a shorthand for listing them all). Here we specify that title,gridProperties(rowCount,columnCount) parametes need to be updated. For example, the following request specifies that the rowCount and columnCount should be updated for the sheet with the given row, column count with the sheet name `Sheet1`.
```

Comment:
You need to specify the properties that need to change in the body of the request. The fields parameter within the requests property should explicitly list those properties (if you want to update all properties, use fields:"*" as a shorthand for listing them all). Here we specify that the title, and gridProperties (rowCount,columnCount) parameters need to be updated. For example, the following request specifies that the rowCount and columnCount of the given row and column of sheet `Sheet1` need to be updated.